insights/parsers/cifs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Get CIFS Info
+=============
+
+This module contains the following parsers:
+
+CIFSDebugData - file ``/proc/fs/cifs/DebugData``
+================================================
+"""
+
+from insights.core import LogFileOutput
+from insights.specs import Specs
+from insights.core.plugins import parser
+
+
+@parser(Specs.cifs_debug_data)
+class CIFSDebugData(LogFileOutput):
+    r"""
+    Parse the ``/proc/fs/cifs/DebugData`` file.
+    Filter is added to the spec.
+
+    Sample input of full file::
+
+        Display Internal CIFS Data Structures for Debugging
+        ---------------------------------------------------
+        CIFS Version 2.41
+        Features: DFS,SMB_DIRECT,STATS,DEBUG,ALLOW_INSECURE_LEGACY,CIFS_POSIX,UPCALL(SPNEGO),XATTR,ACL
+        CIFSMaxBufSize: 16384
+        Active VFS Requests: 0
+
+        Servers:
+        1) ConnectionId: 0x4 Hostname: CVXDFAVB1
+        Number of credits: 512 Dialect 0x302 signed
+        TCP status: 1 Instance: 22
+        Local Users To Server: 1 SecMode: 0x3 Req On Wire: 0
+        In Send: 0 In MaxReq Wait: 0
+        DFS origin full path: \\ad.abc.com\DFSROOT\name1
```

Comment:
From the advisor rule's logic, we want to get the `full path` here for other usage. But if we process the hostname obfuscate for such line, the target full path will be changed, will that still usable for later logics? 

Or, if we use `no_obfuscate=['hostname', 'ip']` for this spec, will that applicable?

Reply:
The hostname here is the remote DFS share provider's hostname. So I think obfuscation won't handle it, right?

Reply:
Yep, currently, only the server's hostname will be obfuscated.

Reply:
Then the `full path` won't be obfuscated.  Thanks @xiangce , @huali027 .